### ISSUE TITLE
Determine team process(es) for completing Sitewide Crew tasks reported within the VA.gov-team repo

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#9969]
## Description
There are many accessibility issues within the va.gov-team repo that would be within the responsibility of the Sitewide Crew teams to resolve.  However, it is currently unclear if these issues are actively getting into the appropriate team backlogs in order to be reviewed, prioritized and ultimately completed. It is possible these issues are becoming stale because they are in a separate repo.

## Acceptance Criteria
- [x] Meet with Public Websites team to determine their process for responding to issues reported within the va.gov-team repo
- [ ] Coordinate with Facilities team to determine their process for responding to issues reported within the va.gov-team repo

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [x] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [ ] `⭐️ Facilities`
- [ ] `⭐️ User support`

**Comment by laflannery** [comment]
@jilladams I am working on this ticket and was wondering if you would be available to discuss in reference to the Public Websites team

**Comment by jilladams** [comment]
Oo yep, this is a gap. We don't currently have a process for scrubbing new tickets to va.gov-team, unless they get assigned to me or Wes. Happy to discuss what we need to do / how to process-ize fixing that.  If you wanna meet with me and @RWEagans together (he's Facilities), that's fine by me - whatever works best.

**Comment by laflannery** [comment]
Meeting separately might be best, I can set up a meeting with you to discuss how Public Websites might want to handle this and I can work with Facilities at another time. Would you want this meeting to include Wes? Or anyone else?

**Comment by laflannery** [comment]
Meeting is scheduled on 8/10 with Public Websites.  Due to other priorities and workload we are going to push a discussion with Facilities out to a later sprint

**Comment by laflannery** [comment]
Notes from Public Websites meeting:

- Current process that is being used:
  - Assign to Jill & Wes
  - This will send a notification
  - Also, at the time of assignment, any critical info or timing should be given to them
  - Jill/Wes can then move the ticket to the va.gov-cms repo, and any necessary tags and put it into the correct pipeline to be refined
- Possible issues with this process:
  - Once issues leave the va.gov-team repo, it leaves the team board
  - Shira had mentioned that she does not want tickets leaving the repo for this reason.  She was specifically discussing collab cycle issues at the time

**Comment by laflannery** [comment]
Next Steps:

- Confirm with Shira if we can move older 508 issues or if she was asking that all issues always remain in the team repo
- Determine when Facilities might have capacity for a discussion
